manifests/server/config.pp

```diff
@@ -89,7 +85,6 @@
       $noauth = true
     }
     if $keyfile and $key {
-      validate_string($key)
       validate_re($key,'.{6}')
```

Comment:
Does this mean it should be ```String[6]``` or ```String[6, 6]```?

Reply:
I think the key is longer than 6 characters, so maybe 6,? I didn't know you could specify the length of the string that way, but that makes sense, can work on replacing this one too.

Reply:
Yeah, I think `String[6]` will do it. Updated.